api/src/main/scala/hmda/api/http/institutions/submissions/SubmissionEditPaths.scala

```diff
@@ -46,18 +46,33 @@ trait SubmissionEditPaths
           completeVerified(institutionId, period, seqNr, uri) {
             val fEditChecks = getValidationState(institutionId, period, seqNr)
 
-            val fSummaryEdits = fEditChecks.map { editChecks =>
-              val s = validationErrorsToEditResults(editChecks.tsSyntactical, editChecks.larSyntactical, Syntactical)
-              val v = validationErrorsToEditResults(editChecks.tsValidity, editChecks.larValidity, Validity)
-              val q = validationErrorsToEditResults(editChecks.tsQuality, editChecks.larQuality, Quality)
-              val m = validationErrorsToMacroResults(editChecks.larMacro)
-              SummaryEditResults(s, v, q, m)
-            }
+            parameters("sortBy".?) {
+              case Some("row") =>
+                val fRowSummary: Future[RowResults] = fEditChecks.map { e =>
+                  val tsErrors = e.tsSyntactical ++ e.tsValidity ++ e.tsQuality
+                  val larErrors = e.larSyntactical ++ e.larValidity ++ e.larQuality
+                  val macroErrors = e.larMacro
+                  validationErrorsToRowResults(tsErrors, larErrors, macroErrors)
```

Comment:
That's true. My thought process was that `validationErrorsToEditResults` set a pattern for how these methods should work, and `validationErrorsToRowResults` should follow it. (though I recognize that there are other ways I could have done it)

If the goal is having the smallest method signature (which isn't necessarily the case), then both methods could just take a `HmdaFileValidationState` (the result of  `getValidationState`) as a parameter. Then the individual conversion methods could do the work of reorganizing that data into the proper format. I'd be happy to make that refactor if that would help